Project.toml

```
@@ -1,7 +1,7 @@
 name = "TimeZones"
 uuid = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
-version = "1.5.2"
+version = "1.5.3"
```

I say this is a bug-fix. It always should have worked

---

I think that makes sense

---

Definitely is a patch but I like to bump versions separate from PRs for this repo.
```suggestion
version = "1.5.3"
```